File: 02_EstructurasControl/Condicionales/Simples/Python/Impuesto_de_la_Renta.py
# === Cálculo de Impuesto a la Renta ===

# Solicitar monto de la renta
renta = float(input("Ingrese el monto de su renta en soles: "))

# Condicional simple
if renta > 4000:
    impuesto = renta * 0.05
    print("El impuesto a pagar es: S/", impuesto)

File: 02_EstructurasControl/Condicionales/Simples/CSharp/Impuesto_de_la_Renta.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Impuesto_a_la_Renta
{
    internal class Program
    {
        static void Main(string[] args)
        {
            Console.WriteLine("=== Cálculo de Impuesto a la Renta ===");
            Console.WriteLine();

            Console.WriteLine("Ingrese el monto de su renta en soles:");
            double renta = Convert.ToDouble(Console.ReadLine());

            // Condicional simple
            if (renta > 4000)
            {
                double impuesto = renta * 0.05;
                Console.WriteLine("El impuesto a pagar es: S/ " + impuesto);
            }


        }
    }
}
